lumigator/python/mzai/sdk/tests/test_health.py

```diff
@@ -0,0 +1,105 @@
+import importlib.resources
+import json
+from http import HTTPStatus
+from pathlib import Path
+
+from pytest import raises
+from requests.exceptions import HTTPError
+
+
+def test_sdk_healthcheck_ok(mock_requests_response, mock_requests, lumi_client):
+    deployment_type = "local"
+    status = "ok"
+    mock_requests_response.status_code = 200
+    mock_requests_response.json = lambda: json.loads(
+        f'{{"deployment_type": "{deployment_type}", "status": "{status}"}}'
+    )
+    check = lumi_client.health.healthcheck()
+    assert check.status == status
+    assert check.deployment_type == deployment_type
+
+
+def test_sdk_healthcheck_server_error(mock_requests_response, mock_requests, lumi_client):
+    mock_requests_response.status_code = 500
+    mock_requests_response.json = lambda: None
+    error = HTTPError(response=mock_requests_response)
+    mock_requests.side_effect = error
+    with raises(HTTPError):
+        lumi_client.health.healthcheck()
+
+
+def test_sdk_healthcheck_missing_deployment(mock_requests_response, mock_requests, lumi_client):
+    status = "ok"
+    mock_requests_response.status_code = 200
+    mock_requests_response.json = lambda: json.loads(f'{{"status": "{status}"}}')
+    check = lumi_client.health.healthcheck()
+    assert check.status == status
+    assert check.deployment_type is None
+
+
+def test_get_deployments_ok(mock_requests_response, mock_requests, lumi_client):
+    mock_requests_response.status_code = 200
+
+    ref = importlib.resources.files("sdk.tests") / "data/deployments.json"
```

Comment:
since we use this resource in several tests, we could add to conftest or at least the top of this module as a fixture

Reply:
The final version does refactor a couple of utility items, would this be more or less your idea?
https://github.com/mozilla-ai/lumigator/blob/b1971690e0b9abf45280c69831f141be7164d468/lumigator/python/mzai/sdk/tests/helpers.py

Reply:
I'd recommend putting those in `conftest`, see examples here: https://github.com/mozilla-ai/lumigator/blob/main/lumigator/python/mzai/evaluator/tests/conftest.py

Reply:
Is this more what you were looking for? 😄 

https://github.com/mozilla-ai/lumigator/pull/250/commits/4dc1c1c4e79f69e2494d638206c9f45a22b33bdb